src/components/Sidebar/index.jsx

```diff
@@ -215,17 +207,20 @@ const SidebarWrapper = styled.div`
     font-weight: 500;
     line-height: 28px;
   }
-
   .page {
     padding: 24px;
 
     background-color: #f1f3f5;
   }
 
+  .frame-list-ui {
+    overflow-y: auto;
+  }
+
```

Comment:
P1: ui라는 말이 대명사와 같은 말이라 ui라는 클래스명을 빼면 좋을 것 같습니다

```suggestion
  .frame-list {
    overflow-y: auto;
  }

```

Reply:
frame-list-ui => frame-list-scroll 로 변경했습니다.
해당 영역은 스크롤바가 추가되는 영역으로 기능 단위를 명사로 붙여 지었습니다.